crunchy.go

```diff
@@ -40,13 +40,16 @@ type Options struct {
 	Hashers []hash.Hash
 	// DictionaryPath contains all the dictionaries that will be parsed (default is /usr/share/dict)
 	DictionaryPath string
+	// Check haveibeenpwned.com database
+	CheckHIBP bool
 }
 
 // NewValidator returns a new password validator with default settings
 func NewValidator() *Validator {
 	return NewValidatorWithOpts(Options{
 		MinDist:        -1,
 		DictionaryPath: "/usr/share/dict",
+		CheckHIBP:	true,
```

Comment:
I think we should keep the default to be `false`, as triggering API calls for every password check is a more invasive change and should be explicitly enabled by the library's user.